client/BaseClient.py

```diff
@@ -236,7 +236,9 @@ def run_claimed_job(self, server, servers, claimed):
         # However, we don't want to hang forever.
         logger.info("Joining ServerUpdater")
         updater_thread.join(self.thread_join_wait)
-        if updater_thread.isAlive():
+        # python >= 3.9 changed Thread.isAlive() -> thread.is_alive()
```

Comment:
Note that python 3.5 and 2.6 both have Thread.is_alive:
https://docs.python.org/2/library/threading.html#threading.Thread.is_alive
https://docs.python.org/3.5/library/threading.html#threading.Thread.is_alive
so unless we need to support python 2.5 or before, I think the check can be removed.

Reply:
Error running with python 3.8:
```
2023-07-10 11:58:19,397:WARNING:Error: Traceback (most recent call last):
  File "C:\Git\opt\civet\client\BaseClient.py", line 257, in run
    self.run_claimed_job(server, [server], claimed)
  File "C:\Git\opt\civet\client\BaseClient.py", line 239, in run_claimed_job
    if updater_thread.isAlive():
AttributeError: 'Thread' object has no attribute 'isAlive'
```